docs/csharp/codedoc.md

```diff
@@ -2,26 +2,1348 @@
 title: Documenting your code
 description: Documenting your code
 keywords: .NET, .NET Core
-author: dotnet-bot
+author: tsolarin
 manager: wpickett
-ms.date: 06/20/2016
+ms.date: 08/18/2016
 ms.topic: article
 ms.prod: .net-core
 ms.technology: .net-core-technologies
 ms.devlang: dotnet
 ms.assetid: 8e75e317-4a55-45f2-a866-e76124171838
 ---
 
-# 🔧 Documenting your code
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/494) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
+# Documenting your code
+
+Documenting C# code is recommended practice as it helps make code easy to understand and maintain.
+Comments are a great way to add documentation to your C# code, they can help explain parts of your code and the great thing is,
+they do not add any overhead to your workflow because they usually are completely ignored by the compiler.
+
+## Single Line Comments
```

Comment:
These first two sections don't contribute to the main focus of the article. I recommend removing them. Below, I point to part of this draft that will be a great opening.